packages/ember-data/lib/system/model/internal-model.js

```diff
@@ -644,6 +646,10 @@ InternalModel.prototype = {
       }
     }
 
+    if (isEmpty(Object.keys(errors))) {
+      this.send('becameInvalid');
+    }
```

Comment:
because it will boule send it (because of events on errors) and it breaks lots of tests